static/src/javascripts/projects/common/views/commercial/survey/survey-adblock.html

```diff
@@ -24,33 +29,53 @@ <h3 class="survey-text__header">
         </h3>
         <div class="survey-content">
             <div class="survey-text">
-                <p class="survey-text__info">
+                <p class="survey-text__strong">
                     <%=surveyText%>
                 </p>
                 <% if (surveyTextSecond) { %>
-                <p class="survey-text__info">
-                    <%=surveyTextSecond%>
-                </p>
+                    <p class="survey-text__strong">
+                        <%=surveyTextSecond%>
+                    </p>
                 <% } %>
                 <% if (surveyTextThird) { %>
                 <p class="survey-text__info">
                     <%=surveyTextThird%>
                 </p>
                 <% } %>
+                <% if (membershipLink || signupLink) { %>
```

Comment:
Better test with `make compile` and `useAssetBundles=true`. In my previous experience, when a lodash template had a test like that, the `text!` plugin created a syntax error in the JS build. But again, I suspect this only happened when my test contained quotes, like `x === 'y'`.